app/Http/Requests/StoreImageAnnotation.php

```diff
@@ -5,7 +5,7 @@
 use Biigle\Image;
 use Biigle\Shape;
 
-class StoreImageAnnotation extends StoreImageAnnotationLabel
+class StoreImageAnnotation extends StoreImageAnnotationLabelFeatureVector
```

Comment:
No need for inheritance here, just implement the stuff of `StoreImageAnnotationLabelFeatureVector` directly in this class.